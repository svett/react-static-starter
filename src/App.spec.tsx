import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

it('renders learn react link', () => {
  const tree = shallow(<App />);
  expect(tree).toMatchSnapshot();
});
