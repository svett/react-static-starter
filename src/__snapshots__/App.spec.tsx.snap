// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src={
        Object {
          "default": Object {},
        }
      }
    />
    <h1>
      welcome to react-static starter kit
    </h1>
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <span>
      Learn
       
      <a
        className="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        React
      </a>
       
      and
       
      <a
        className="App-link"
        href="https://react-static.js.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        React Static
      </a>
    </span>
  </header>
</div>
`;
